MaxiOps/injectorScheduling/arbitrum/Migrate-USDC-V1-injector-to-V2.json

```diff
@@ -0,0 +1,191 @@
+            "name": "doNotStartBeforeTimestamp",
+            "type": "uint56"
+          }
+        ],
+        "name": "addRecipients",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "recipients": "[0x4be0f45db66e2e6271b52549f26e69771c7dfc71, 0x9419c8b7ff370db3823164acea6d9664f4b9c8a9]",
+        "amountPerPeriod": "1000000000",
+        "maxPeriods": "5",
+        "doNotStartBeforeTimestamp": "1740070800"
+      }
+    },
+    {
+      "to": "0xfa7b21B30325DBbd4A71ee2B2EDE74A7d8A2c0E4",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address[]",
+            "name": "recipients",
+            "type": "address[]"
+          },
+          {
+            "internalType": "uint256",
+            "name": "amountPerPeriod",
+            "type": "uint256"
+          },
+          { "internalType": "uint8", "name": "maxPeriods", "type": "uint8" },
+          {
+            "internalType": "uint56",
+            "name": "doNotStartBeforeTimestamp",
+            "type": "uint56"
+          }
+        ],
+        "name": "addRecipients",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "recipients": "[0x4945a8dedaee4f70b4d6967691ed6ae365a9446f, 0xfb7a403481c0bfcb7c3848a1f8f848f0f7d61cd4]",
+        "amountPerPeriod": "500000000",
+        "maxPeriods": "5",
+        "doNotStartBeforeTimestamp": "1740070800"
+      }
+    },
+    {
+      "to": "0xfa7b21B30325DBbd4A71ee2B2EDE74A7d8A2c0E4",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address[]",
+            "name": "recipients",
+            "type": "address[]"
+          },
+          {
+            "internalType": "uint256",
+            "name": "amountPerPeriod",
+            "type": "uint256"
+          },
+          { "internalType": "uint8", "name": "maxPeriods", "type": "uint8" },
+          {
+            "internalType": "uint56",
+            "name": "doNotStartBeforeTimestamp",
+            "type": "uint56"
+          }
+        ],
+        "name": "addRecipients",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "recipients": "[0x59b889b00e12c52fc0bbfebe297fe64975273715]",
+        "amountPerPeriod": "750000000",
+        "maxPeriods": "5",
+        "doNotStartBeforeTimestamp": "1740070800"
+      }
+    },
+    {
+      "to": "0x37D9dC70bfcd8BC77Ec2858836B923c560E891D1",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "uint256", "name": "id", "type": "uint256" }
+        ],
+        "name": "cancelUpkeep",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "signers": "",
+        "transmitters": "",
+        "f": "",
+        "onchainConfigBytes": "",
+        "offchainConfigVersion": "",
+        "offchainConfig": "",
```

Comment:
i think all of these args are redundant and can be left out

the method's signature is `cancelUpkeep(id)`